content/docs/misc/token-refresher-proxy/_index.md

```diff
@@ -0,0 +1,144 @@
+---
+date: '2023-08-01'
+title: Patch token-refresher to use a cluster proxy
+tags: ["OSD", "ROSA"]
+authors:
+  - Andy Repton
+---
+
+Currently, if you deploy a ROSA or OSD cluster with a proxy, the token-refresher pod in the openshift-monitoring namespace will be in crashloopbackoff. There is an RFE open to resolve this, but until then this can affect the ability of the cluster to report telemetry and potentially update. This article provides a workaround on how to patch the token-refresher deployment until that RFE has been fixed using the patch-operator.
+
+Thanks to Paul Czarkowski for the original idea.
+
+## Prerequisites
+
+* ROSA or OSD Cluster deployed using a cluster wide proxy
+
+## Problem Demo
+
+### Deploy a cluster with proxy settings like so:
+
+```bash
+$ rosa create cluster --http-proxy http://your.proxy:8080 --https-proxy https://your.secure.proxy:8080 --no-proxy my.local.domain
+```
+
+### Check your pods in the openshift-monitoring namespace
+
+```bash
+$ oc get pods -n openshift-monitoring | grep token-refresher
+token-refresher-74ff5d9f96-2mm4v                         1/1     CrashLoopBackOff     5             5m
+```
+
+## Install the patch-operator from the console
+
+### Log into your cluster as a user able to install Operators and browse to Operator Hub
+![install patch operator](/docs/misc/token-refresher-proxy/images/install-patch-operator.png)
+
+Accept the warning about community operators:
+
+![accept community warning](/docs/misc/token-refresher-proxy/images/accept-community-operator.png)
+
+### Install the Patch Operator
+
+![install patch operator](/docs/misc/token-refresher-proxy/images/accept-community-operator-2.png)
+
+## Create the patch
+### Create the service account and cluster role required for the patch operator to function
+
```

Comment:
each of these should be in an ordered list with a description of what is being done.